app/forms/state_file/nj_homeowner_property_tax_worksheet_form.rb

```diff
@@ -0,0 +1,13 @@
+module StateFile
+  class NjHomeownerPropertyTaxWorksheetForm < QuestionsForm
+    set_attributes_for :intake,
+                       :property_tax_paid
+
```

Comment:
[DUST] We have to allow this to be empty, since the UX must allow someone to press "continue" if they want to skip claiming the deduction. 

This is what it looks like when empty.
<img width="685" alt="image" src="https://github.com/user-attachments/assets/90b0339f-76b9-4d5e-8fdd-908681cf51d1" />

This is what it looks like if I write 0.
<img width="700" alt="image" src="https://github.com/user-attachments/assets/c16caa3d-6cd9-4011-91af-64070f1dcd2a" />

And when I try to submit.
<img width="733" alt="image" src="https://github.com/user-attachments/assets/504737c9-d38f-46c6-aca1-ca04e37b84fe" />

As a user, I don't know how I'd be expected to know to remove the 0 that I wrote in before clicking continue.

Reply:
I feel like we *should* allow 0 in this case, and it should be treated the same as empty.

Seems like a design/product question to confirm?

We should still validate against negative though.

Reply:
Right, a negative value here would break calculations down the line. Do you know if 0 and nil get treated any differently for rent_paid and property_tax_paid?

Reply:
Confirmed with design and product that we should accept 0 but treat it on the backend as if it's `nil`